xml/templates/message_templates/contribution_invoice_receipt_html.tpl

```diff
@@ -234,9 +234,9 @@
         </tr>
         <tr>
           {if $organization_name}
-            <td style="padding-left:17px;"><font size="1" align="center">{$display_name}  ({$organization_name})</font></td>
+            <td style="padding-left:17px;"><font size="1" align="center">{contact.display_name}  ({$organization_name})</font></td>
```

Comment:
I didn't do organization name because in my testing it was only showing up for organizations not individuals which meant it was repeated - but I'll test that further

Reply:
Tangential, but since you mention it - I use invoicing a lot, and never found why the organization name was repeated. if an invoice is "care of Some Person", then it would be in the address fields.

Reply:
hmm -  good point